catalyst-gateway/bin/src/jinja.rs

```diff
@@ -0,0 +1,48 @@
+//! `minijinja` static global variables
+
+use std::sync::LazyLock;
+
+use minijinja::{Environment, Template};
+
+use crate::db::event::signed_docs::{
+    ADVANCED_SELECT_SIGNED_DOCS_TEMPLATE, SELECT_SIGNED_DOCS_TEMPLATE,
+};
+
+/// Jinja template source struct.
+pub(crate) struct JinjaTemplateSource {
+    /// Jinja template name
+    pub(crate) name: &'static str,
+    /// Jinja template source
+    pub(crate) source: &'static str,
+}
+
+/// Global static `minijinja::Environment` with all preloaded templates
+#[allow(clippy::unwrap_used)]
+static JINJA_ENV: LazyLock<Environment> = LazyLock::new(|| {
+    let mut env = minijinja::Environment::new();
+
+    // Preload templates
+    env.add_template(
+        SELECT_SIGNED_DOCS_TEMPLATE.name,
+        SELECT_SIGNED_DOCS_TEMPLATE.source,
+    )
+    .unwrap();
+    env.add_template(
+        ADVANCED_SELECT_SIGNED_DOCS_TEMPLATE.name,
+        ADVANCED_SELECT_SIGNED_DOCS_TEMPLATE.source,
+    )
+    .unwrap();
+
+    env
+});
+
+/// Initializes `JINJA_ENV`
+pub(crate) fn init_jinja() {
```

Comment:
What is this for?  It would init the first time `get_template` is called.

Reply:
> It would init the first time get_template is called.
Yes, it is correct. But to avoid initialisation of the `jinja` environment and loading all templates, I've added this function and called it in this place 
https://github.com/input-output-hk/catalyst-voices/blob/b088b06310308aec73bcd91e9c784c3d4b1a9217/catalyst-gateway/bin/src/cli.rs#L41